src/prisma/schema.prisma

```diff
@@ -1155,6 +1159,29 @@ model ProposalBlock {
   @@index([createdBy])
 }
 
+model RewardBlock {
+  id        String
+  deletedAt DateTime?
+  createdAt DateTime  @default(now())
+  createdBy String    @db.Uuid
+  updatedAt DateTime  @default(now())
+  updatedBy String    @db.Uuid
+  spaceId   String    @db.Uuid
+  parentId  String
+  rootId    String    @db.Uuid
+  schema    Int
+  type      String    @default("board")
```

Comment:
not sure if we have a type default for other block tables, but i feel like it would be better to remove the default and make it required